pandas/compat/__init__.py

```diff
@@ -20,7 +20,7 @@
     np_version_under1p20,
 )
 from pandas.compat.pyarrow import (
-    pa_version_under1p0,
+    pa_version_under1p01,
```

Comment:
If the min version of pyarrow is 1.0.1, it shouldn't be possible for this to ever be true. I think ``import_optional_dependency`` raises for us if the version is too old

Reply:
This variable is used to check pyarrow installation. Removing this would involve more changes to the codebase. I'll look into it more later today.

Reply:
you can just leave the name of the variable and change the version in the check